examples/hello-world/styles.css

```
@@ -1,15 +1,119 @@
+  --card-bg: rgba(255, 255, 255, 0.95);
+  --card-border: rgba(226, 232, 240, 0.8);
+  --text-primary: #f8fafc;
+  --text-secondary: #94a3b8;
+  --shadow-lg: 0 10px 15px -3px rgba(0, 0, 0, 0.3);
+  --shadow-xl: 0 20px 25px -5px rgba(0, 0, 0, 0.4);
 }
 
-body {
+* {
+  box-sizing: border-box;
+  font-family: 'Inter', -apple-system, sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)